doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -261,17 +291,27 @@ given widths appropriately.
 Temporal Filtering
 ..................
 
-All previous filters operate on images, before conversion to voxel signals.
+Rather than optimizing spatial properties of the neuroimaging data,
+the user typically wants to improve aspects of temporal data properties,
+before conversion to voxel signals.
 :class:`NiftiMasker` can also process voxel signals. Here are the possibilities:
 
 - Confound removal. Two ways of removing confounds are provided. Any linear
-  trend can be removed by activating the `detrend` option. It is not activated
-  by default in :class:`NiftiMasker` but is almost essential. More complex confounds can
+  trend can be removed by activating the `detrend` option.
+  This accounts for slowly (as opposed to abruptly or transiently) changes
```

Comment:
Also why did you remove the note about detrend=False by default although it is probably a good idea to set it to True in general ?